README.md

```diff
@@ -273,8 +276,13 @@ objectives:
 - **value** *numeric*, required field, used to compare values gathered from
   metric source. Only needed when using a `thresholdMetric`.
 
+Either `target` or `targetPercent` must be used.
+
 - **target** *numeric [0.0, 1.0)*, required, budget target for given objective
-  of the SLO
+  of the SLO. A `target: 0.9995` is equivalent to `targetPercent: 99.95`.
```

Comment:
Should we add a comment to `target` and `targetPercent` in the description about choosing one or the other?

At the moment it appears they're both required, unless someone sees the note on line 256?

Reply:
Updated descriptions of both to make it explicit how they must be used.